core/src/apps/bitcoin/get_address.py

```diff
@@ -2,11 +2,12 @@
 from trezor.messages import InputScriptType
 from trezor.messages.Address import Address
 
+from .sign_tx import addresses
```

Comment:
all non-signers (here `addresses.py`, `multisig.py`) should be moved to `apps/bitcoin`, seeing as the other operations are also using them

Reply:
Fixed in 4497ea8a957a16c2bc9885a88c7831c55872b0bd

Reply:
Not really, same goes for writers.py

Reply:
Should be done, now.